android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/ReceiptCapturePlugin.kt

```diff
@@ -33,112 +34,91 @@ import kotlinx.coroutines.async
 class ReceiptCapturePlugin : Plugin() {
     private val receiptCapture = ReceiptCapture()
 
+    /**
+     * Initializes the receipt capture functionality.
+     *
+     * This method initializes the receipt capture service, preparing it for use. You should call this method
+     * before using any other receipt capture features.
+     *
+     * @param call The Capacitor plugin call instance.
+     */
     @PluginMethod
-    fun initialize(call: PluginCall) = receiptCapture.initialize(call, context)
+    fun initialize(call: PluginCall) = receiptCapture.initialize(call, activity)
 
```

Comment:
Login and logout needs a better explanation of how it works. The different scenarios to login or logout from email or retailer accounts